src/azure/ApiCenter/ApiCenterRestAPIs.ts

```diff
@@ -1,7 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 export const APICenterRestAPIs = {
+    GetResourceGroup: (subscriptionId: string, resourceGroupName: string, restApiVersion: string) => `https://management.azure.com/subscriptions/${subscriptionId}/resourcegroups/${resourceGroupName}?api-version=${restApiVersion}`,
     GetAPIService: (subscriptionId: string, resourceGroupName: string, apiCenterName: string, restApiVersion: string) => `https://management.azure.com/subscriptions/${subscriptionId}/resourceGroups/${resourceGroupName}/providers/Microsoft.ApiCenter/services/${apiCenterName}?api-version=${restApiVersion}`,
+    listApiCenterServers: (subscriptionID: string, restApiVersion: string) => `https://management.azure.com/subscriptions/${subscriptionID}/providers/Microsoft.ApiCenter?api-version=${restApiVersion}`,
```

Comment:
I think it is not to list from ApiCenterServers, it is get metadata from ApiCenter ARM?

Reply:
update to `GetApiCenterMetadataFromSub` and relative type turn to `SubApiCenterMetaData`